mdal/api/mdal.h

```diff
@@ -647,6 +665,8 @@ MDAL_EXPORT MDAL_DatasetH MDAL_G_addDataset(
  * \param verticalExtrusion Double Array holding the vertical level values for the voxels
  *               Size must be Face count + Volume count
  * \returns empty pointer if not possible to create dataset (e.g. group opened in read mode), otherwise handle to new dataset
+ *
+ * \since MDAL 0.9.0
```

Comment:
this is added in this version?

Reply:
I think that is a copy/paste error. I will correct it.

Reply:
I have to correct myself - that is in the comments which are new in version 0.9.0 but in a previous PR. I just noticed that the version tag was not added in that PR